docs/prober-and-monitoring.md

```diff
@@ -0,0 +1,102 @@
+# Prober and monitoring
+
+**PMAP is not a official Google product.**
+
+Monitoring consists of two parts: prober and alerting.
+
+## Prober
+
+[Prober](../prober/) is a tool that constantly probes pmap services to check if
+the services are up. It is built as a go binary and deployed to [cloud run
+job](https://cloud.google.com/run/docs/overview/what-is-cloud-run#jobs), and is
```

Comment:
Nits: `It is built as a go binary, deployed as [cloud run
job](https://cloud.google.com/run/docs/overview/what-is-cloud-run#jobs), and 
triggered by [cloud scheduler](https://cloud.google.com/scheduler).`

Reply:
Added